From 02d0d0a20873a9e42e30113f3e31bce011793f30 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Mon, 12 Jun 2017 14:47:06 +0200
Subject: [PATCH] udev-rules: pull in 40-redhat-cpu-hotplug.rules

Cpu hotplug policy was split off to separate file. Let's pull in the new
rules file by default.

See: https://bugzilla.redhat.com/show_bug.cgi?id=1266322

Resolves: #1460707
---
 modules.d/95udev-rules/module-setup.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/95udev-rules/module-setup.sh b/modules.d/95udev-rules/module-setup.sh
index e0f2e381..13cafc2a 100755
--- a/modules.d/95udev-rules/module-setup.sh
+++ b/modules.d/95udev-rules/module-setup.sh
@@ -27,6 +27,7 @@ install() {
     fi
 
     inst_rules \
+        40-redhat-cpu-hotplug.rules \
         40-redhat.rules \
         50-firmware.rules \
         50-udev-default.rules \
